### DISCUSSION TITLE
Unable to reverify University

### DISCUSSION THREAD

**Discussion by pedrogsc** [community/community#50794] [GitHub Education]
### Select Topic Area

Question

### Body

I'm trying to renew my teacher application, but when I select the school email address I receive the following prompt:

_We require applicants to use a school-issued email address to apply._

My university is the University of Coimbra, Portugal
My email address is *******@icnas.uc.pt

My application was accepted the first time in February.

Please, if you can help me.

Thank you.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @pedrogsc ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by pedrogsc** [reply]
Sorry, was unaware. Thank you!

**Comment by samus-aran** [comment]
Hi @pedrogsc 

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting a [similar comment/discussion in multiple places.](https://github.com/orgs/community/discussions/51143) As a result, we have locked and closed this discussion and we ask that you no longer cross-post. 👍 

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
